docker/scripts/setup_cnf.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash 
+
+mongodb1=`getent hosts ${MONGO1} | awk '{ print $1 }'`
+mongodb2=`getent hosts ${MONGO2} | awk '{ print $1 }'`
+mongodb3=`getent hosts ${MONGO3} | awk '{ print $1 }'`
+arbiter=`getent hosts ${ARBITER} | awk '{ print $1 }'`
+
+port=${PORT:-27017}
+
+echo "Waiting for startup.."
+until mongo --host ${mongodb1}:${port} --eval 'quit(db.runCommand({ ping: 1 }).ok ? 0 : 2)' &>/dev/null; do
+  printf '.'
+  sleep 1
+done
+
+echo "Started.."
+
+echo setup.sh time now: `date +"%T" `
+mongo --host ${mongodb1}:${port} <<EOF
+   var cfg = {
+        "_id": "${RS}",
+        "protocolVersion": 1,
+        "configsvr": true,
```

Comment:
This is a good catch.

I noticed that things work without it for MongoDB 4.0 to 5.0 without it. The replica set can figure out it is a config server replica set. Since documentation explicitly asks for it, we should add it. Better to not depend on any undocumented functionality.